code/modules/research/anomaly/anomaly_core.dm

```diff
@@ -61,3 +61,9 @@
 	desc = "The neutralized core of a vortex anomaly. It won't sit still, as if some invisible force is acting on it. It'd probably be valuable for research."
 	icon_state = "vortex_core"
 	anomaly_type = /obj/effect/anomaly/bhole
+
+/obj/item/assembly/signaler/anomaly/hallucination
+	name = "\improper hallucination anomaly core"
+	desc = "The neutralized core of a hallucination anomaly. It seems to be moving, but it's probably your imagination. It'd probably be valuable for research."
+	icon_state = "anomaly_core"
```

Comment:
make sprites for these

Reply:
perhaps make them GAGsified for easier sprites for the future ones too

Reply:
why you want me to make sprites 😢

Reply:
GAGS sprite would be pretty easy since they're all just recolors

Reply:
yeah i'm not going to gagsify them as of now, i'll add the needed hallucination sprites